dace/sdfg/utils.py

```diff
@@ -1847,6 +1847,11 @@ def get_global_memlet_path_src(sdfg: SDFG, state: SDFGState, edge: MultiConnecto
         if len(pedges) > 0:
             pedge = pedges[0]
             return get_global_memlet_path_src(psdfg, pstate, pedge)
+        else:
```

Comment:
Should we do this for dst too?

Reply:
I don't think that the inverse is possible, i.e., if something is written in an inner scope, it should not be possible that it is only read in an outer scope.